Microsoft.HealthVault/Helpers/Validator.cs

```diff
@@ -63,6 +72,27 @@ public static void ThrowIfArgumentNull(object argument, string argumentName, str
             }
         }
 
+        public static void ThrowIfGuidEmpty(Guid argument, string argumentName)
+        {
+            if (argument == null)
```

Comment:
Haha good catch. The whole reason I made that function was because checking null wouldn't work, then I forgot to actually implement it

---
In reply to: [116864858](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/34#discussion_r116864858) [](ancestors = 116864858)